screens/map/MapScreen.js

```diff
@@ -20,6 +20,9 @@ import {
   SearchBar,
 } from '../../styled/store';
 
+const minSnapPoint = 300;
```

Comment:
Also this should be a little higher. In Figma you should be able to peek the images https://www.figma.com/file/drhxZht9Mvogrc6aLn7MsK/Spring-20-DC-Central-Kitchen?node-id=7827%3A55

Reply:
Changed `minSnapPoint` to `325`, which is a little higher (16px vs 11px in figma) but given that our current product cards do not have backgrounds, I gave it a little bump to show at least some parts of the product image. Feel free to change it to something that makes more sense in terms of design decision!